composer.json

```diff
@@ -38,21 +37,16 @@
     },
     "suggest": {
         "ext-curl": "If you use CURLRequest class",
-        "ext-dom": "If you use TestResponse",
         "ext-exif": "If you run Image class tests",
-        "ext-fileinfo": "Improves mime type detection for files",
         "ext-gd": "If you use Image class GDHandler",
         "ext-imagick": "If you use Image class ImageMagickHandler",
-        "ext-libxml": "If you use TestResponse",
         "ext-memcache": "If you use Cache class MemcachedHandler with Memcache",
         "ext-memcached": "If you use Cache class MemcachedHandler with Memcached",
         "ext-mysqli": "If you use MySQL",
         "ext-oci8": "If you use Oracle Database",
         "ext-pgsql": "If you use PostgreSQL",
         "ext-readline": "Improves CLI::input() usability",
         "ext-redis": "If you use Cache class RedisHandler",
-        "ext-simplexml": "If you format XML",
```

Comment:
This extension requires the [libxml](https://www.php.net/manual/en/book.libxml.php) PHP extension.